resen/resencmd.py

```diff
@@ -190,29 +174,52 @@ def do_status(self,args):
 
     def do_start_jupyter(self,args):
         """Usage:
->>> start_jupyter bucket_name local_port bucket_port\t: Start a jupyter notebook server on port bucket_port available at local_port.
->>> start_jupyter bucket_name local_port bucket_port --lab\t: Start a jupyter lab server on port bucket_port available at local_port.
+>>> start_jupyter bucket_name
         """
         inputs,num_inputs = self.parse_args(args)
-        lab = False
-        if num_inputs == 3:
+
+        if num_inputs == 1:
             pass
-        elif num_inputs == 4:
-            if inputs[3][0] == '-':
-                if inputs[3] == '--lab':
-                    lab = True
-                else:
-                    print("Syntax Error. See 'help start_jupyter'.")
-                    return
         else:
             print("Syntax Error. See 'help start_jupyter'.")
             return
 
+
+        # get bucket name from input
         bucket_name = inputs[0]
-        local_port = int(inputs[1])
-        bucket_port = int(inputs[2])
 
-        status = self.program.start_jupyter(bucket_name,local_port,bucket_port,lab=lab)
+        # get bucket infomrmation (ports and status)
+        # This stuff may be better suited to exist in some kind of "status query" inside of Resen.py
+        ind = self.program.bucket_manager.bucket_names.index(bucket_name)
```

Comment:
Here's the issue with the `start_jupyter` command.  Checking to make sure that a bucket name exists happens in `self.program.start_jupyter()` on line 208, which is consistent with what's going on with `stop_jupyter`, but we're trying to get the port information up here.

Reply:
done